src/components/About/icons/Values/Group/index.tsx

```diff
@@ -0,0 +1,43 @@
+const GroupIcon = ({ width = 96, height = 76, className = "" }) => {
+  return (
+    <svg
+      width={width}
+      height={height}
+      viewBox="0 0 96 76"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+      className={`${className}`}
+    >
+      <path
```

Comment:
For the `aria-labelledby=` attribute from above, we'll need these defined inside the `<svg/>` element. These are are only a suggestion though, we probably don't need both or we don't need them at all. The contents of `<title/>` and `<desc/>` are just placeholders right now.

```suggestion
      <title id="group-icon-title">Group Icon</title>
      <desc id="gropu-icon-description">
        A Group Icon representing a group of some kind.
      </desc>
      <path
```